tools/android_lint/project.xml

```diff
@@ -47,6 +47,7 @@
     <src file="../../../flutter/shell/platform/android/io/flutter/embedding/android/FlutterImageView.java" />
     <src file="../../../flutter/shell/platform/android/io/flutter/embedding/android/KeyData.java" />
     <src file="../../../flutter/shell/platform/android/io/flutter/embedding/android/FlutterActivityLaunchConfigs.java" />
+    <src file="../../../flutter/shell/platform/android/io/flutter/embedding/engine/renderer/SurfaceTextureSurfaceProducer.java" />
```

Comment:
Not a blocker but do you know if there is a better way for us to be linting than manually including each source file? Were these files just forgotten, and therefore not having linting enforced?

Reply:
Actually it looks like the inclusion is not "manual", we have a script for it, but still people need to run the script? And if they don't do that when adding new files, we won't enforce linting in ci? Is that correct?

Reply:
The script runs on CI - this only got updated because I ran it locally.